Sources/GDSCommon/UI/UIKit/SharedViews/Buttons/ButtonViewModel.swift

```diff
@@ -2,11 +2,16 @@ import UIKit
 
 public protocol ButtonViewModel {
     var title: GDSLocalisedString { get }
-    var icon: String? { get }
+    var icon: ButtonIconViewModel? { get }
```

Comment:
What's the process for this, or do we have one yet?

Reply:
Chatted to @amopatel1234, we agreed there is no way to make this a non breaking change. Only thing we can do here is ensure that we have the versioning set up ASAP, so that we can avoid this in the future.